test/JDBC/expected/BABEL-4926.out

```diff
@@ -0,0 +1,225 @@
+create index IDX_REPRODUCTION on BABEL_4926_REPO(babel_4926_col1,babel_4926_Col2,babel_4926_COL);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute where attname like '%babel_4926%' and attrelid = (select oid from pg_class where relname = 'babel_4926_repo') order by attname asc;
+go
+~~START~~
+varchar#!#varchar
+babel_4926_col#!#{bbf_original_name=babel_4926_COL}
+babel_4926_col1#!#<NULL>
+babel_4926_col2#!#{bbf_original_name=babel_4926_Col2}
+~~END~~
+
+-- dropping the table BABEL_4926
+drop table BABEL_4926;
+go
+-- dropping the table BABEL_4926_REPO
+drop table BABEL_4926_REPO;
+go
+
+-- create a new table BABEL_4926 with changing the name of column while making new table 5
+create table BABEL_4926(babel_4926_COL int);
+go
+select babel_4926_COL from BABEL_4926;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table BABEL_4926_REPO with column babel_4926_COL
+select babel_4926_COL as babel_4926_COl1 into BABEL_4926_REPO from BABEL_4926;
+go
+-- Indexing over column babel_4926_COL1
+create index IDX_REPRODUCTION on BABEL_4926_REPO(babel_4926_COl1);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute where attname like '%babel_4926%' and attrelid = (select oid from pg_class where relname = 'babel_4926_repo') order by attname asc;
+go
+~~START~~
+varchar#!#varchar
+babel_4926_col1#!#{bbf_original_name=babel_4926_COl1}
+~~END~~
+
+-- dropping the table BABEL_4926
+drop table BABEL_4926;
+go
+-- dropping the table BABEL_4926_REPO
+drop table BABEL_4926_REPO;
+go
+-- create a new table BABEL_4926 with changing the name of column while making new table(testing compatibility of function query in select into statement) 6
+create table BABEL_4926(babel_4926_COL int);
+go
+select babel_4926_COL from BABEL_4926;
+go
+~~START~~
+int
+~~END~~
+
+-- create a new function to double the value
+CREATE FUNCTION dbo.babel_4926_double_function (@in INT) RETURNS INT
+AS
+BEGIN
+    RETURN (2 * @in);
+END;
+go
+-- Create a new table BABEL_4926_REPO with column babel_4926_COL1 using function
+select babel_4926_double_function(babel_4926_COL) as babel_4926_COL1 into BABEL_4926_REPO from BABEL_4926;
+go
+-- Indexing over column babel_4926_COL1
+create index IDX_REPRODUCTION on BABEL_4926_REPO(babel_4926_COl1);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute where attname like '%babel_4926%' and attrelid = (select oid from pg_class where relname = 'babel_4926_repo') order by attname asc;
+go
+~~START~~
+varchar#!#varchar
+babel_4926_col1#!#{bbf_original_name=babel_4926_COL1}
+~~END~~
+
+-- dropping the table BABEL_4926
+drop table BABEL_4926;
+go
+-- dropping the table BABEL_4926_REPO
+drop table BABEL_4926_REPO;
+go
+drop function babel_4926_double_function;
+go
+
+-- create a new table BABEL_4926 7
+create table BABEL_4926([babel_4926_COL$] int);
+go
+select [babel_4926_COL$] from BABEL_4926; 
+go
+~~START~~
+int
+~~END~~
+
```

Comment:
Let's also add a tests with column names having length larger than 64 chars, special characters, quoted with `[]` and `""` etc.